doc/NIM_Response_Table.md

```diff
@@ -0,0 +1,29 @@
+# Northern Images (NIM) Response Table
+The following two tables contains all of the possible responses and error codes from the NIM payload.
```

Comment:
Should specify that error codes are received from the satellite as negative numbers

Reply:
Okay fair enough